sympy/physics/control/lti.py

```diff
@@ -24,19 +34,48 @@ def var(self):
         return self._var
 
     def __add__(self, other):
-        pass
+        if not self.var == other.var:
+            raise ValueError("Last argument for both Transfer Functions must be same.")
```

Comment:
This message should be about the variable, not the arguments of the TF. 

"Both the Transfer Functions should have the same variable" or both should be anchored with the same variable. Something like that.